ports/esp8266/README.md

```diff
@@ -22,21 +22,30 @@ Supported features include:
 - Modules for HTTP, MQTT, many other formats and protocols via
   https://github.com/micropython/micropython-lib .
 
-Work-in-progress documentation is available at
-http://docs.micropython.org/en/latest/esp8266/ .
+Documentation is available at http://docs.micropython.org/en/latest/esp8266/quickref.html.
 
 Build instructions
 ------------------
 
-The tool chain required for the build is the OpenSource ESP SDK, which can be
-found at <https://github.com/pfalcon/esp-open-sdk>.  Clone this repository and
-run `make` in its directory to build and install the SDK locally.  Make sure
-to add toolchain bin directory to your PATH.  Read esp-open-sdk's README for
-additional important information on toolchain setup.
+You need the esp-open-sdk toolchain (which provides both the compiler and libraries), which
+you can obtain using one of the following two options:
+
+ - Use a Docker image with a pre-built toolchain (**recommended**).
+   To use this, install Docker, then prepend
+   `docker run --rm -v $HOME:$HOME -u $UID -w $PWD larsks/esp-open-sdk ` to the start
```

Comment:
I'd suggest really spelling this out step by step for newer users.
(you'd think newer users wouldn't be trying to build from scratch, but a surprising number of people do)

Reply:
OK, will do.

Yes it's really useful for people who want to freeze modules, so definitely worth spelling it out in detail.

Reply:
this was left as-is; please open a new PR for additional changes (if any)

Reply:
Yes, please - more detail here would be *very* helpful.  It seems like several steps are skipped here.